src/app/views/main-header/settings/ThemeChooserV9.tsx

```diff
@@ -0,0 +1,68 @@
+import { Menu, MenuTrigger, MenuButton, MenuPopover, MenuList, MenuItem } from '@fluentui/react-components';
+import { useAppDispatch, useAppSelector } from '../../../../store';
+import { componentNames, eventTypes, telemetry } from '../../../../telemetry';
+import { PopupsComponent } from '../../../services/context/popups-context';
+import { changeTheme } from '../../../services/slices/theme.slice';
+import { BrightnessHighRegular, WeatherMoonFilled, CircleHalfFillFilled } from '@fluentui/react-icons';
+
+const availableThemes = [
+  {
+    key: 'light',
+    displayName: 'Web Light',
+    icon: <BrightnessHighRegular />
+  },
+  {
+    key: 'dark',
+    displayName: 'Web Dark',
+    icon: <WeatherMoonFilled />
+  },
+  {
+    key: 'high-contrast',
+    displayName: 'Teams High Contrast',
+    icon: <CircleHalfFillFilled />
+  }
+];
+
+const ThemeChooserV9: React.FC<PopupsComponent<null>> = () => {
+  const dispatch = useAppDispatch();
+  const appTheme = useAppSelector(state=> state.theme);
+
+
+  const handleChangeTheme = (selectedTheme: { key: string; displayName: string; icon: JSX.Element }) => {
+    const newTheme: string = selectedTheme?.key ?? '';
+    // Applies the theme to the Fluent UI components
+    switch (newTheme) {
+    case 'light':
+      return dispatch(changeTheme('light'));
+    case 'dark':
+      return dispatch(changeTheme('dark'));
+    case 'high-contrast':
+      return dispatch(changeTheme('high-contrast'));
+    }
+    telemetry.trackEvent(eventTypes.BUTTON_CLICK_EVENT, {
+      ComponentName: componentNames.SELECT_THEME_BUTTON,
+      SelectedTheme: newTheme.replace('-', ' ').toSentenceCase()
+    });
+  };
+  return (
+    <Menu>
```

Comment:
I think a better component to use here is a [Radio Group](https://react.fluentui.dev/?path=/docs/components-radiogroup--docs#layout--horizontal).

Also, I noticed that the theming does not stick. If I choose web light, when I reload, it goes back to the initial one. The theme should be persisted on local storage and remain after reloads. (happens in current GE so it's a good bug to fix on the go)

Lastly, I've seen this on theme choosers where there's an option to "use system default". Here we already check the default system them and we can have it as the default selected. Also, we can add an event listener that captures changes to the default system theme and applies them appropriately.

Reply:
Nice catch! Yeah we can definitely use the default system theme here. Will make this change.

Reply:
I could definitely use the Radio Group/ Radio component, customize the component to use slots for the icons.